.goreleaser.yml

```diff
@@ -57,17 +57,10 @@ checksum:
   name_template: '{{ .ProjectName }}_v{{ .Version }}_SHA256SUMS'
   algorithm: sha256
 signs:
-  - artifacts: checksum
-    args:
-      # if you are using this is in a GitHub action or some other automated pipeline, you
-      # need to pass the batch flag to indicate its not interactive.
-      - "--batch"
-      - "--local-user"
-      - "{{ .Env.GPG_FINGERPRINT }}"
-      - "--output"
-      - "${signature}"
-      - "--detach-sign"
-      - "${artifact}"
+  - cmd: signore
+    args: ["sign", "--dearmor", "--file", "${artifact}", "--out", "${signature}"]
+    artifacts: checksum
+    signature: ${artifact}.sig
```

Comment:
Ehm, I wonder if packer init will work with sig files🤔.

We would need to add a checksummer type here:

https://github.com/hashicorp/packer/blob/a766d59d82c129a40f8612e648cde2d051a91fba/command/init.go#L73-L75

Reply:
Goreleaser will generate the SHA256SUM checksum file. The signore project will sign the generated SHA256SUM file and create the .sig file, which is binary. The created binary file is the same as if we were using the gpg command except signore is a hosted service for signing. 

Looking at the checksum code for Packer init, I see it reads the file and then parses the checksum against the hash for the plugin. If we wanted to check the sig file we would want to include a gpg verify process into the init step. This is not needed at this time. In the future it might be worth looking into.

Reply:
Ah, I see, super cool, thanks for the explanation ! LGTM then !!